tests/data_utils.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+import json
+import os
+import platform
+
+from future.utils import old_div
+
+from quantized_mesh_tile import TerrainTopology
+from quantized_mesh_tile.editable_terrain import EditableTerrainTile
+from quantized_mesh_tile.global_geodetic import GlobalGeodetic
+
+MAX = 32767.0
+
+
+def lerp(p, q, time):
+    return ((1.0 - time) * p) + (time * q)
```

Comment:
why duplicate this function?

Reply:
there is no really good reason, will be deleted.
It was left over as i moved (refactor) some helper functions  like buildTerrainTile to  data_utils.